Unix/doc/setup-ntlm-omi.md

```diff
@@ -264,3 +265,11 @@ channels.
 
 WinRM and powershell settings TBD
 
+### Cause 5: Could not connect
+
+You might get the error "Could not connect" when you run `omicli` command to remote server.
+
+- First, you need to check if you enable http port or https port in /etc/opt/omi/conf/omiserver.conf.
```

Comment:
Change wording: `First, check if you enabled the HTTP or HTTPS port in /etc/opt/omi/conf/omiserver.conf.`